application/controllers/timers.py

```diff
@@ -0,0 +1,90 @@
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
+from application.models.timer import Timer
+from application import db
+
+timer_fields = {
+    'id': fields.Integer,
+    'work_interval': fields.String,
+    'rest_interval': fields.String
+}
+
+timer_list_fields = {
+    'count': fields.Integer,
+    'timers': fields.List(fields.Nested(timer_fields)),
+}
+
+timer_post_parser = reqparse.RequestParser()
+timer_post_parser.add_argument(
+    'work_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='work_interval parameter is required'
+)
+timer_post_parser.add_argument(
+    'rest_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='rest_interval parameter is required'
+)
+
+class TimersResource(Resource):
```

Comment:
Just trying to wrap my head around how Timers is going to relate to us on the FE. Are you envisioning that when we eventually have users, that one Timer would be associated with a user, meaning it's like "Timer settings" or if it instead represents one cycle of a POM and a user would have multiple "Timers" if that makes sense at all.